index.html

```diff
@@ -147,6 +147,11 @@ <h2>
       <pre class="idl" data-cite="HR-TIME">
         [Exposed=Window, SecureContext]
         interface Gamepad {
```

Comment:
I think this needs to then become an: 

```suggestion
        interface Gamepad : EventTarget {
```